### DISCUSSION TITLE
请问muduo中的http可以解析multipart/form-data格式的数据吗，也就是浏览器上传文件(例如图片）时会拆成多个数据包发送

### DISCUSSION THREAD

**Discussion by xt1234567** [chenshuo/muduo#576] [General]
GitHub issues are for tracking bugs, not for general discussing like a forum.

If you have a general question to ask, send it to mailing list:

muduo-library@googlegroups.com

Or visit:
https://groups.google.com/forum/#!forum/muduo-library

When file an issue muduo, please provide a [SSCCE](http://sscce.org):
Short, Self Contained, Correct (Compilable), Example.

If you can't compile muduo, make sure you install `cmake` and `boost` from the
official package repository, e.g. `apt` or `yum`, before opening a bug.
Don't open a bug if you installed boost from a third-party source or
downloaded it by yourself, and couldn't compile muduo, thank you.

Also specify the exact environment where the issue occurs:

**Comment by chenshuo** [comment]
muduo 的 Http 不是用来写 web server，而只是为 server 提供一个暴露调试信息的窗口。
